README.md

```diff
@@ -195,6 +215,23 @@ Count number of events in each queue:
 jq 'map_values(map(keys[0]))' queue_dump.json
 ```
 
+### Debug console
+
+If the configuration option `console.enabled` is set to `true`, a unix socket named `debug.socket` by default can be found next to the configuration while the node is running. It can be connected to by tools like `socat`:
+
+```sh
+$ socat - unix:/path/to/debug.socket
+```
+
+Entering `help` will show available commands. The interface can also be scripted, it may be helpful to change some of the per-connection settings in this case:
```

Comment:
```suggestion
Entering `help` will show available commands. 

The interface can also be scripted, it may be helpful to change some of the per-connection settings in this case:
```

Reply:
I didn't want the single line there to feel so lonely! In my mind, this makes it a little bit awkward. I can still change it if you insist though, I don't exactly have a strong opinion on this :P